### PR TITLE
Updating docs and copier question

### PR DESCRIPTION
**Pull request by drewoldag**
Updated two copier questions:

1. Include the name of the isort package explicitly, and update documentation about that.
2. Add git-lfs question to the list of "custom" configuration questions. 

Updated the new_project.rst docs in an effort to be a little more streamlined around the `pip install -e .` part.

### CODE REVIEW

File: python-project-template/docs/{% if include_notebooks %}notebooks.rst{% endif %} [hunk 1]
```diff
@@ -3,4 +3,4 @@ Notebooks
 
 .. toctree::
 
-    Introducing Jupyter Notebooks <nb/intro_notebook>
\ No newline at end of file
+    Introducing Jupyter Notebooks <notebooks/intro_notebook>
```
Comment (OliviaLynn) [inline python-project-template/docs/{% if include_notebooks %}notebooks.rst{% endif %}:6]: Nice catch, thanks!

File: docs/source/new_project.rst [hunk 2]
```diff
@@ -32,7 +32,7 @@ Copier will ask you questions for how to set up the project. These questions wil
      - The license type you want to use for this project. Options are MIT and BSD. For more information on these options see `Github's license page <https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository>`_.
    * - *What tooling would you like to use to enforce code style?*
      - A linter is a tool to automatically format for consistency (see :doc:`Linting <../practices/linting>`). We provide options for `black <https://black.readthedocs.io/en/stable/>`_, `pylint <https://pypi.org/project/pylint/>`_, or no linter. Choosing a linter will include it as a project dependency and include it in the :doc:`pre-commit <../practices/precommit>` hooks. Defaults to ``pylint`` during simple installation. 
-   * - *Do you want to use a tool to maintain a specific ordering for module imports?*
+   * - *Do you want to use a isort maintain a specific ordering for module imports?*
```
Comment (OliviaLynn) [inline docs/source/new_project.rst:35]: I think the 'a' of 'use a tool' slipped in here

**Review by OliviaLynn — APPROVED** [review]
This looks great!

File: docs/source/new_project.rst (hunk 2 above)
Comment (drewoldag) [reply docs/source/new_project.rst]: Oops, fixed.

**Pull request merged**